app/models/beta_signup.rb

```diff
@@ -0,0 +1,5 @@
+class BetaSignup < ActiveRecord::Base
+  belongs_to :user
+
+  validates_uniqueness_of :user_id
```

Comment:
Prefer the new style validations `validates :column, uniqueness: value` over `validates_uniqueness_of`.